package.json

```diff
@@ -138,6 +138,7 @@
   "resolutions": {
     "@commitlint/is-ignored/semver": "^7.3.5",
     "@microsoft/eslint-plugin-sdl/eslint-plugin-react": "^7.26.0",
+    "@react-native-community/cli-tools/shell-quote": "^1.7.3",
```

Comment:
interesting - do we need to do this because it's not used by CLI v8 or something like that?

Reply:
It's fixed upstream, but we have to override it here since it depends on 1.6.1 explicitly. I'm overriding `@react-native-community/cli-tools` explicitly to remember to remove it when we bump to 0.68.